lib/peeps.rb

```diff
@@ -0,0 +1,33 @@
+require 'pg'
+require 'date'
+
+class Peeps
+
+  attr_reader :message, :timestamp
+
+  def initialize(message, timestamp)
+    @message = message
+    @timestamp = Time.parse(timestamp).strftime('%I:%M %p %b %-d, %Y ')
+  end
+
+  def self.all
```

Comment:
Looks like you've met the requirement to show peeps in reverse chronological order but not tested it